src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/dashboard/CreateDashboardScenarioTest.java

```diff
@@ -32,7 +33,7 @@ void should_create_scenario() throws Exception {
 
         UUID caseId = UUID.randomUUID();
         String hearingFeeByTime = "4 pm";
-        OffsetDateTime hearingFeeByDate = OffsetDateTime.now();
+        LocalDate hearingFeeByDate = OffsetDateTime.now().toLocalDate();
```

Comment:
I made this change as test sometimes fails on the milliseconds, and we don't need time for the date anyway.